sinks/dashboard/items/can_sender/can_sender.py

```diff
@@ -0,0 +1,217 @@
+from pyqtgraph.Qt import QtCore
+from pyqtgraph.Qt import QtGui
+from pyqtgraph.Qt import QtWidgets
+from omnibus import Sender
+from parsers import publisher
+from ..registry import Register
+from ..dashboard_item import DashboardItem
+from .canlib_metadata import CanlibMetadata
+import sources.parsley.message_types as mt
+
+@Register
+class CanSender(DashboardItem):
+    """
+    QGridLayout blueprint
+    |--------|----|----|----|----|------|
+    | (0,0)  | lbl| lbl|... |lbl |(0,10)|
+    |--------|----|----|----|----|------|
+    |Msg Type|Byte|Byte|... |Byte|Button|
+    |--------|----|----|----|----|------|
+    | (2,0)  | lbl| lbl|... |lbl |(2,10)|
+    |--------|----|----|----|----|------|
+    Msg type = CAN bus message type
+    Byte     = CAN bus data
+    lbl      = describes the type of data in that column
+    """
+    __pulse_occurances = 3
+    __pulse_frequency = 100
+    __number_of_bytes = 8
+
+    def __init__(self, props=None):
+        super().__init__()
+        self.props = props
+
+        self.layout_manager = QtWidgets.QGridLayout(self)
+        self.setLayout(self.layout_manager)
+
+        self.canlib_info = CanlibMetadata("can_sender_data.txt")
+        self.sender_thing = Sender()
```

Comment:
I am god awful at naming variables and the name ```self.sender``` conflicts with Sender(), this probably needs a better name